### DISCUSSION TITLE
Icarus click events

### DISCUSSION THREAD

**Discussion by ChuckPliskin** [primefaces/community#1077] [PrimeFaces Templates] [answered]
Good morning,
I purchased the icarus template for Primefaces and i have a problem with the topbar menu. Norhing happen when i click on any item.
I'm developing on Jakarta 10.0, Primefaces 13.0.5 and Tomcat 10.1
Can i have some help please ?
Thank you !

**Comment by melloware** [comment, marked as answer]
Hmmm it is not happening on the Icarus showcase: https://www.primefaces.org/icarus/

Are there any errors in the JS Console?
